sw/airborne/modules/computer_vision/opticflow/opticflow_calculator.c

```diff
@@ -307,23 +315,52 @@ bool calc_fast9_lukas_kanade(struct opticflow_t *opticflow, struct image_t *img,
   } else if (!opticflow->feature_management) {
     // needs to be set to 0 because result is now static
     result->corner_cnt = 0;
-    // FAST corner detection
-    // TODO: There is something wrong with fast9_detect destabilizing FPS. This problem is reduced with putting min_distance
-    // to 0 (see defines), however a more permanent solution should be considered
-    fast9_detect(&opticflow->prev_img_gray, opticflow->fast9_threshold, opticflow->fast9_min_distance,
-                 opticflow->fast9_padding, opticflow->fast9_padding, &result->corner_cnt,
-                 &opticflow->fast9_rsize,
-                 &opticflow->fast9_ret_corners,
-                 NULL);
+
+    if(CORNER_METHOD == EXHAUSTIVE_FAST) {
+      // FAST corner detection
+      // TODO: There is something wrong with fast9_detect destabilizing FPS. This problem is reduced with putting min_distance
+      // to 0 (see defines), however a more permanent solution should be considered
+      fast9_detect(&opticflow->prev_img_gray, opticflow->fast9_threshold, opticflow->fast9_min_distance,
+                   opticflow->fast9_padding, opticflow->fast9_padding, &result->corner_cnt,
+                   &opticflow->fast9_rsize,
+                   &opticflow->fast9_ret_corners,
+                   NULL);
+
+    }
+    else if (CORNER_METHOD == ACT_FAST) {
+        // ACT-FAST corner detection:
+        // TODO: all relevant things should be settings:
+        float long_step = 10; // 5
+        float short_step = 2; // 2
+        int min_gradient = 10; // 10
+        printf("opticflow->fast9_threshold = %d, n_agents = %d, n_time_steps = %d\n", opticflow->fast9_threshold, n_agents, n_time_steps);
```

Comment:
Remove or make optional

Reply:
made them settings and removed the printf